docs/commands/lint.md

```diff
@@ -2,239 +2,378 @@
+    features.openapi: []
 ```
 
-In this case, after resolving the path behind the `main` name (see the `Configuration file` tab), `lint` will validate the `definition.json` file. The presence of the Redocly configuration file is mandatory.
+ In this example, we only want to validate files under `main`:
 
-#### Empty entrypoints
+```bash Command
+redocly lint main
+```
 
-You can omit entrypoints completely when executing the `lint` command.
+##### Method 3: Pass no entrypoints
 
-```bash Command
-openapi lint
+Running `lint` without specifying a particular API definition file checks all definition files in your project directory.
+
+```bash
+redocly lint
 ```
 
-```yaml Configuration file
-apis:
-  main:
-    root: ./openapi/definition.json
-  production:
-    root: ./openapi/production.yaml
-  sandbox:
-    root: ./openapi/sandbox.yaml
+:::warning Important
+If you run `redocly lint` without entrypoints, and your project has no configuration files, you'll get an error.
+:::
+
+### How to reference a custom config file
+
+By default, Redocly CLI looks for the [Redocly configuration file](/docs/cli/configuration/configuration-file.mdx) in the current working or project directory. Using the optional `--config` argument provides an alternative path to a different configuration file if one exists.
+
+```bash
+redocly lint --config=./another/directory/anotherconfig.yaml
 ```
 
-In this case, if no API definitions are specified, `lint` validates all entrypoints listed under `apis` in your Redocly configuration file. The presence of the configuration file is mandatory.
+### How to activate rule sets
 
-:::warning Important
+Redocly CLI comes with [built-in rules](./resources/built-in-rules.md) that are used to validate API definitions. If you want to override a rule or reduce its severity, you can do so by modifying `redocly.yaml` (globally or per API definition).
 
-If you try to execute the `lint` command without entrypoints when your project doesn't have any configuration files, the `lint` command will display an error.
+An alternative approach is to use **rule sets** to quickly invoke a lot of rules at the same time.
 
-:::
+Redocly CLI has three rule sets, each specifying different levels of severity, that can be applied to API definition file validation by adding the `--extends` option to the `lint` command:
+
+* For you rebels out there, the `minimal` rule set will apply the least severe config for each built-in rule, and deactivate others entirely. We like to call this our death-metal rule set. View the GitHub Source [here](https://github.com/Redocly/openapi-cli/blob/ed997e2586e7adb7e32d8107cac79c452891f765/packages/core/src/config/minimal.ts).
+* Late 80s easy-listening more your jam? Then our `recommended` rule set is for you. It will apply the recommended config for each built-in rule. View the GitHub source [here](https://github.com/Redocly/openapi-cli/blob/ed997e2586e7adb7e32d8107cac79c452891f765/packages/core/src/config/recommended.ts).
```

Comment:
```suggestion
* Late 80s easy-listening more your jam? Then our `recommended` configuration is for you. It will apply the recommended config for each built-in rule. View the GitHub source [here](https://github.com/Redocly/openapi-cli/blob/ed997e2586e7adb7e32d8107cac79c452891f765/packages/core/src/config/recommended.ts).
```